GOVERNANCE.md

```diff
@@ -0,0 +1,93 @@
+# Cozystack Governance
+
+This document defines project governance for Cozystack.
+
+## Overview
+
+**Cozystack**, a Cloud Native Computing Foundation (CNCF) project, is committed
+to building an open, inclusive, productive, and self-governing open source
+community focused on building a high-quality cloud native registry.
```

Comment:
@tym83 Cozystack is not a cloud-native registry. I suppose, it should be "Free PaaS platform and framework for building clouds"?